src/invoices/InvoicesList/InvoicesList.js

```diff
@@ -73,14 +74,16 @@ class InvoicesList extends Component {
           initialResultCount={INITIAL_RESULT_COUNT}
           resultCountIncrement={RESULT_COUNT_INCREMENT}
           visibleColumns={visibleColumns}
-          viewRecordComponent={noop}
+          viewRecordComponent={Invoice}
           onSelectRow={onSelectRow}
+          finishedResourceName="perms"
```

Comment:
are you sure about this?

> finishedResourceName | string | Newly created records are displayed as soon as they are created. Usually that is as soon as the record itself exists, but in some cases it is not until some other operation has completed -- for example, new user records are not ready to be displayed until their permissions have been added. In such situations, this property may be set to the name of the resource which must have completed its operations before the record is ready.

Reply:
these 2 properties should be removed as I had a problem with opening details pane and tried to add smth from another modules - forgot about them

and seems we need to review proprs for SearchAndSort in all our modules